content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -255,6 +261,13 @@ $ kubectl label namespace default istio.io/dataplane-mode=ambient
 Congratulations! You have successfully added all pods in the default namespace
 to the mesh. Note that you did not have to restart or redeploy anything!
 
+Check once again the ztunnel logs for the proxy has received the network namespace (netns) information about an ambient application pod, and has started proxying for it:
+
+{{< text bash >}}
+$ kubectl logs ds/ztunnel -n istio-system | grep "starting proxy"
+received netns, starting proxy
```

Comment:
Is this really the expected output? Isn't there text before this string?
```suggestion
... received netns, starting proxy
```

Reply:
right looks better with the `...` I originally wanted to showcase the whole entry but the `_verify_contains` and `_verify_like` just were not working for the type of output